lib/membrane/rtp/vad.ex

```diff
@@ -108,8 +111,21 @@ defmodule Membrane.RTP.VAD do
 
   @impl true
   def handle_process(:input, %Membrane.Buffer{} = buffer, _ctx, state) do
-    vad_extension = Enum.find(buffer.metadata.rtp.extensions, &(&1.identifier == 1))
-    <<_v::1, level::7>> = vad_extension.data
+    {extension, buffer} = Header.Extension.pop_identifier(buffer, state.header_extension_id)
```

Comment:
My bad, when I look at it in this scenario `Header.Extension.pop` makes more sense than `pop_identifier`... Can you bring it back? 😢

Reply:
No worries, it's back